src/Settings/components/keepAwake.jsx

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import { ListItem, Switch, Avatar } from "@rneui/themed";
+import { useSelector, useDispatch } from "react-redux";
+import { nightModeStyles, nightModeColor } from "../styles/nightModeStyles";
+import STRINGS from "../../common/localization";
+import { toggleScreenAwake } from "../../common/actions";
+
+function KeepAwake() {
+  const dispatch = useDispatch();
+  const { isNightMode, isScreenAwake, isAutoScroll } = useSelector((state) => state);
+  const { containerNightStyles } = nightModeStyles(isNightMode);
+  const nightColor = nightModeColor(isNightMode);
+  const { KEEP_AWAKE } = STRINGS;
+  const screenIcon = require("../../../images/screenonicon.png");
+  return (
+    <ListItem bottomDivider containerStyle={containerNightStyles}>
+      <Avatar source={screenIcon} />
```

Comment:
Missing alt attribute in avatar component.

Reply:
Veer ji, the Avatar component doesn't include an 'alt' attribute. I can add 'title' if you say so.